testsuite/forge-cli/src/main.rs

```diff
@@ -649,6 +650,33 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
             },
             false,
         ),
+        "twin_validator_test" => config
+            .with_network_tests(vec![&TwinValidatorTest])
+            .with_initial_validator_count(NonZeroUsize::new(20).unwrap())
+            .with_initial_fullnode_count(5)
+            .with_genesis_helm_config_fn(Arc::new(|helm_values| {
+                helm_values["chain"]["epoch_duration_secs"] = 300.into();
+            }))
+            .with_success_criteria(SuccessCriteria::new(
+                if duration.as_secs() > 1200 {
```

Comment:
I would just hardcode the duration in `FORGE_RUNNER_DURATION_SECS` and have a fixed threshold instead.